src/commands/apps/create.ts

```diff
@@ -46,6 +47,12 @@ export default class AppCreateCommand extends Command {
   @hasArg
   platform: string;
 
+  @help("The environment of the app. This must be one word, alphanumeric, first letter capitalized.")
+  @shortName("e")
+  @longName("environment-app")
```

Comment:
This should be reversed: "app-environment" or "release-type"

Reply:
I think `app-environment` is probably acceptable. @blparr please object if you have an issue with that :)